improver/wxcode/weather_symbols.py

```diff
@@ -544,6 +550,11 @@ def create_symbol_cube(self, cubes: Union[List[Cube], CubeList]) -> Cube:
             optional_attributes.update(
                 update_model_id_attr_attribute(cubes, self.model_id_attr)
             )
+        if self.record_run_attr is not None and self.model_id_attr is not None:
```

Comment:
Again, consider what will happen if `record_run_attr` is set without `model_id_attr`.

Reply:
See above.